File: app/src/main/java/com/akexorcist/localizationapp/viewpager/TwoFragment.java
package com.akexorcist.localizationapp.viewpager;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.akexorcist.localizationapp.R;

/**
 * Created by Akexorcist on 7/22/15 AD.
 */
public class TwoFragment extends Fragment {

    public static Fragment newInstance() {
        return new TwoFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_two, container, false);
    }
}

File: app/src/main/java/com/akexorcist/localizationapp/viewpager/TwoFragment.kt
package com.akexorcist.localizationapp.viewpager

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.akexorcist.localizationapp.databinding.FragmentViewPagerTwoBinding

class TwoFragment : Fragment() {
    companion object {
        @JvmStatic
        fun newInstance(): Fragment = TwoFragment()
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        val binding = FragmentViewPagerTwoBinding.inflate(layoutInflater, container, false)
        return binding.root
    }
}
